app/helpers/articles_helper.rb

```diff
@@ -0,0 +1,20 @@
+module ArticlesHelper
+  def article_element_data(tag_type, contents)
+    if(tag_type == 'img')
+      picture = Picture.find(contents)
+      image_tag picture.image.url, class: 'img-responsive'
+    else
+      elements = {
+        sub_head: {
+          actual_tag: :h3,
+          raw_data: contents
+        },
+        description: {
+          actual_tag: :p,
+          raw_data: contents
+        }
+      }
+      content_tag(elements[tag_type.to_sym][:actual_tag], elements[tag_type.to_sym][:raw_data])
```

Comment:
__[rubocop]__
* Line is too long. [95/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)